qemu/tests/cfg/qemu_img_info_while_running.cfg

```diff
@@ -0,0 +1,20 @@
+- qemu_img_info_while_vm_running:
+    virt_test_type = qemu
+    kill_vm = yes
+    start_vm = no
+    force_create_image = no
+    image_size = 20G
```

Comment:
Not suggest to give this value at here unless it is not configurable for this test.